spec/pluralrules.html

```diff
@@ -53,6 +53,82 @@
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-GetOperands" aoid="GetOperands">
+       <h1>GetOperands (x)</h1>
+       <p>
+         When the GetOperands abstract operation is called with argument _x_ (which must be a finite Number value or a String representation of it), it returns an Object value with the following list of plural operands used by plural rules functions.
+       </p>
+
+       <emu-table id="table-plural-operands">
+         <emu-caption>Plural Rules Operands</emu-caption>
+         <table class="real-table">
+           <thead>
+             <tr>
+               <th>Symbol</th>
+               <th>Value</th>
+             </tr>
+           </thead>
+           <tr>
+             <td>n</td>
+             <td>absolute value of the source number (integer and decimals)</td>
+           </tr>
+           <tr>
+             <td>i</td>
+             <td>integer digits of n.</td>
+           </tr>
+           <tr>
+             <td>v</td>
+             <td>number of visible fraction digits in n, <i>with</i> trailing zeros.</td>
+           </tr>
+           <tr>
+             <td>w</td>
+             <td>number of visible fraction digits in n, <i>without</i> trailing zeros.</td>
+           </tr>
+           <tr>
+             <td>f</td>
+             <td>visible fractional digits in n, <i>with</i> trailing zeros.</td>
+           </tr>
+           <tr>
+             <td>t</td>
+             <td>visible fractional digits in n, <i>without</i> trailing zeros. </td>
+           </tr>
+         </table>
+       </emu-table>
+
+       <p>
+        The following steps are taken:
+       </p>
+
+       <emu-alg aoid="GetOperands">
+         1. Let _sv_ be ? ToString(_x_).
+         1. If _sv_ starts with *"-"*, then
```

Comment:
sort of, the reality is that in this method we don't care about negative numbers, we just need to make sure that we use a positive value, isn't it? If that's true, then something like:

```
1. Let _nv_ be ? ToNumber(_x_).
1. If _nv_ < 0, set _nv_ to -_nv_.
```

then do all computations based on the string value of _nv_ :)